### ISSUE TITLE
Add automatic Clean Up functionality

### ISSUE THREAD

**Issue by kerofr** [ribbons/RadioDownloader#208]
Not a problem, but a request.  Would it be possible to include in the Clean Up options a choice for "Remove Downloads which: Are older than "x" days" so that we can specify deleting files that are say... 6 days old?  This would give us the option to make the process of cleanup automatic after a certain amount of days.
Thanks for your consideration.
Kent

**Comment by ribbons** [comment]
Sorry for the slow reply, I was on holiday.

Yes, this sounds like something that would be good to have in the application - just to make sure I've got it clear, you are talking about enhancing the *Clean Up* options so that they run automatically if the user desires?

We'd need to think carefully about the UI to make sure that people don't activate the feature accidentally but I'd be happy to accept a pull-request that implemented this in a clear and user-friendly way.

**Comment by kerofr** [comment]
Yes, that's what I'm thinking. Juice Portable has this function in the feed properties.  They use a check box to opt in.  Thanks for having a look at this.
